ironfish-graph-explorer/src/data.json

```diff
@@ -0,0 +1,155 @@
+[
+  {
+    "hash": "a1ba91bd54fcfe97d0df8fe1f66b6217197b96cd52db97fad58b275e164600ab",
```

Comment:
Should this be deleted?

Reply:
Let's keep it as a small demo so they don't have to export to see it working